README.md

```diff
@@ -39,8 +39,8 @@ clean postfix installation.
 
 **WARNING**: If you specify `previous: replaced`, the role reinstalls the postfix
 package and replaces the existing `/etc/postfix/main.cf` and
-`/etc/postix/master.cf` files. Ensure to back up those files to preserve your
-settings.
+`/etc/postfix/master.cf` files. <!--- wokeignore:rule=master -->
```

Comment:
@nhosoi do you know if there is a way to have an "allow" rule in woke.yml?  For example, it would be good to be able to specify something like
`
- term: /etc/postfix/master.cf
  severity: allow
- term: master
  severity: error
```
that is, if `woke` found the string `/etc/postfix/master.cf` in any file, it would be allowed, no warnings.  But other instances of the string `master` would be flagged.

Reply:
`severity` takes the value `info`, as well. But it's also "counted" and reported as a failure in the woke.yml action unless we apply this patch https://github.com/get-woke/woke/pull/252... (;_;)

Reply:
Plus, if both `master` and `/etc/postfix/master.cf` is defined in the rule, it seems both matches.

Rule example:
```
rules:
  - name: master.cf
    terms:
      - master.cf
    alternatives:
      - master.cf
    severity: info

  - name: master
    terms:
      - master
    alternatives:
      - primary
      - source
      - initiator,requester
      - controller,host
      - director
    severity: warning
```
Woke result.
```
README.md:42:13-22: `master.cf` may be insensitive, use `master.cf` instead (info)
`/etc/postix/master.cf` files. Ensure to back up those files to preserve your
             ^
README.md:42:13-19: `master` may be insensitive, use `primary`, `source`, `initiator,requester`, `controller,host`, `director` instead (warning)
`/etc/postix/master.cf` files. Ensure to back up those files to preserve your
             ^
```

Reply:
Do we want to add another value `allow` to `severity` which completely skips the check???

Reply:
> Do we want to add another value `allow` to `severity` which completely skips the check???

Something to consider for a future version of `woke`, but for now, this is ok.